selfdrive/test/test_routes.py

```diff
@@ -86,6 +86,7 @@
   TestRoute("49f3c13141b6bc87|2021-07-28--08-05-13", HYUNDAI.KONA_HEV),
   TestRoute("5dddcbca6eb66c62|2020-07-26--13-24-19", HYUNDAI.KIA_STINGER),
   TestRoute("d624b3d19adce635|2020-08-01--14-59-12", HYUNDAI.VELOSTER),
+  TestRoute("007d5e4ad9f86d13|2021-09-25--11-23-14", HYUNDAI.KIA_K5_2021),
```

Comment:
this route isn't fully uploaded. connect the device to wifi, then go to https://useradmin.comma.ai/?onebox=007d5e4ad9f86d13%7C2021-09-25--11-23-14 and click "upload all logs" once it pops up

Reply:
hmm, this car fingerprints as a sonata

Reply:
Just got home. Yes I changed the fingerprint and FW of the Sonata to the K5 so OP would work. I don't see an "upload all logs" in my drive log .

Reply:
hmm, looks like those files got deleted. I'm guessing you factory resetted or uninstalled openpilot. we'll need another route, so find a more recent one that has the "upload all logs" button

Reply:
ok

Reply:
If you comment the new route here and keep the device connected to wifi, I'll get the PR updated and route uploaded

Reply:
just uploaded  007d5e4ad9f86d13|2021-09-30--15-09-23

Reply:
Test route updated in new commit.